ivy/functional/frontends/numpy/mathematical_functions/extrema_finding.py

```diff
@@ -260,3 +260,25 @@ def _fmax(
     if ivy.is_array(where):
         ret = ivy.where(where, ret, ivy.default(out, ivy.zeros_like(ret)), out=out)
     return ret
+
+@handle_numpy_out
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def _fmin(
```

Comment:
are you also implementing support for this function? if yes, may I know if you have a GitHub issue for this?

Reply:
Yes i have created here is the link :-https://github.com/unifyai/ivy/pull/13625